roles/ceph-ansible/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+# list of devices which will be used for Ceph.
+# If it is empty, list is autodetected by ceph-ansible
+devices: []
+
+# use collocated journals? Default to true for simplicity.
+collocation: true
+
+# if collocation is false and journal devices are set,
+# create journals on journal devices.
```

Comment:
If ``devices`` variable is empty devices are autodetected by ceph-ansible. If ``collocation`` is ``false``,  ``journal_devices`` should contain list of devices for journals.